guides/v2.3/graphql/develop/create-graphqls-file.md

```diff
@@ -77,7 +77,7 @@ input ProductSortInput {
 
 `ProductSortInput` indicates that the attributes are available to catalog (Product) queries. If you specify a module-specific value such as `VolumizerSortInput`, then the attribute will be available only to queries processed by that module.
 
-`SortEnum` is defined in the base `schema.graphqls` file.
+`SortEnum` defines the order of results returned (ASC or DESC). For the definition see [`schema.graphqls`](https://github.com/magento/magento2/blob/2.3/app/code/Magento/GraphQl/etc/schema.graphqls) file.
```

Comment:
change the first part of the description to: "indicates whether to return results in ascending or descending order."

Reply:
@erikmarr done!